upgrade_test.py

```diff
@@ -209,6 +210,9 @@ def _upgrade_node(self, node, upgrade_sstables=True, new_scylla_repo=None, new_v
         if self.params.get("enable_tablets_on_upgrade"):
             scylla_yaml_updates.update({"experimental_features": ["tablets", "consistent-topology-changes"]})
 
+        if self.params.get("enable_raft_topology"):
```

Comment:
I understand this is temporary measure (according to commit message this should be default for 6.0) - please post a comment with issue so it can be removed when complete.

Reply:
rewritten